helpers/emojis/getEmojiUrl.ts

```diff
@@ -1,6 +1,6 @@
 import { Bot } from "../../bot.ts";
 
 /** Creates a url to the emoji from the Discord CDN. */
-export function emojiUrl(bot: Bot, id: bigint, animated = false) {
+export function getEmojiURL(_bot: Bot, id: bigint, animated = false): string {
```

Comment:
Hmm renaming this is a bit weird as there is no fetching involved here. The get makes me think there was some fetching going on here

Reply:
I thought that adding the get keyword to the file/function names that were missing them would make them more consistent.

Reply:
Consistency is good but not if it causes confusion. `get` should be reserved for prefixing on functions that make a request to discord. Otherwise its just a util function and imo something like this can in fact be removed from `helpers` and moved to utils folder